lib/cdo/honeybadger.rb

```diff
@@ -84,4 +84,30 @@ def self.parse_exception_dump(error)
 
     [error_message, error_lines]
   end
+
+  # Returns all issues (across cronjobs, dashboard, and pegasus) that have occured since midnight.
+  # @return [Array[Hash]] An array of hashes summarizing the recent issues.
+  def self.get_recent_issues
+    issues = []
+
+    midnight_epoch = Time.now.to_i / 86400 * 86400
```

Comment:
:D I usually assume it's not my job until 9:30, but to be fair I also follow up the next morning to see if any HB errors occurred - I don't know if everyone's doing that, and I don't use the DOTD script for it right now, I use my inbox.  Doubled awareness of overnight errors doesn't seem like a bad thing.